Client/Frontend/Theme/DarkTheme.swift

```diff
@@ -4,19 +4,150 @@
+//    override var disabledTint: UIColor { return UIColor.Photon.Grey30 }
+}
+
+fileprivate class DarkLoadingBarColor: LoadingBarColor {
+//    override var start: UIColor { return UIColor.Photon.Blue50A30 }
+//    override var end: UIColor { return UIColor.Photon.Blue50 }
+}
+
+fileprivate class DarkTabTrayColor: TabTrayColor {
+    override var tabTitleText: UIColor { return defaultTextAndTint }
+    override var tabTitleBlur: UIBlurEffect.Style { return UIBlurEffect.Style.dark }
+    override var background: UIColor { return UIColor.Photon.Ink90 }
+    override var cellBackground: UIColor { return defaultBackground }
+    override var toolbar: UIColor { return UIColor.Photon.Ink80 }
+    override var toolbarButtonTint: UIColor { return defaultTextAndTint }
+    override var cellCloseButton: UIColor { return defaultTextAndTint }
+//    override var privateModeLearnMore: UIColor { return UIColor.Photon.Purple60 }
+//    override var privateModePurple: UIColor { return UIColor.Defaults.MobilePrivatePurple }
+//    override var privateModeButtonOffTint: UIColor { return toolbarButtonTint }
+//    override var privateModeButtonOnTint: UIColor { return UIColor.Photon.Grey10 }
+    override var cellTitleBackground: UIColor { return UIColor.Photon.Ink70 }
+}
+
+fileprivate class DarkTopTabsColor: TopTabsColor {
+    override var background: UIColor { return UIColor.Photon.Grey80 }
+    override var tabBackgroundSelected: UIColor { return UIColor.Photon.Grey80 }
+    override var tabBackgroundUnselected: UIColor { return UIColor.Photon.Ink80 }
+    override var tabForegroundSelected: UIColor { return UIColor.Photon.Grey10 }
+    override var tabForegroundUnselected: UIColor { return UIColor.Photon.Grey40 }
+   
+//    override var selectedLineNormalMode: UIColor { return UIColor.Photon.Blue60 }
+//    override var selectedLinePrivateMode: UIColor { return UIColor.Photon.Purple50 }
+//    override var buttonTint: UIColor { return .red }
+//    override var privateModeButtonOffTint: UIColor { return buttonTint }
+//    override var privateModeButtonOnTint: UIColor { return UIColor.Photon.Grey10 }
+    override var closeButtonSelectedTab: UIColor { return tabForegroundSelected }
+    override var closeButtonUnselectedTab: UIColor { return tabForegroundUnselected }
+    override var separator: UIColor { return UIColor.Photon.Grey50 }
+}
+
+fileprivate class DarkTextFieldColor: TextFieldColor {
+    override var background: UIColor { return UIColor.Photon.Ink70 }
+    override var textAndTint: UIColor { return defaultTextAndTint }
+   // override var separator: UIColor { return defaultSeparator }
+}
+
+fileprivate class DarkHomePanelColor: HomePanelColor {
+    override var toolbarBackground: UIColor { return defaultBackground }
+    override var toolbarHighlight: UIColor { return UIColor.Photon.Blue50 }
+    override var toolbarTint: UIColor { return UIColor.Photon.Grey30 }
+    override var panelBackground: UIColor { return UIColor.black }
```

Comment:
This should really be a Photon colour, but they don't seem to have black. Perhaps you can open an issue requesting it… 🙂

Reply:
Why?

Reply:
Because (as I understand it) we would like to use black because it looks nice, and we would like to have the colours we're using in the Design System.